test/functional/rpc_psbt.py

```diff
@@ -413,14 +412,12 @@ def run_test(self):
         # Create outputs to nodes 1 and 2
         node1_addr = self.nodes[1].getnewaddress()
         node2_addr = self.nodes[2].getnewaddress()
-        txid1 = self.nodes[0].sendtoaddress(node1_addr, 13)
-        txid2 = self.nodes[0].sendtoaddress(node2_addr, 13)
-        blockhash = self.generate(self.nodes[0], 6)[0]
-        vout1 = find_output(self.nodes[1], txid1, 13, blockhash=blockhash)
-        vout2 = find_output(self.nodes[2], txid2, 13, blockhash=blockhash)
+        utxo1 = self.create_outpoints(node=self.nodes[0], outputs=[{node1_addr: 13}])[0]
+        utxo2 = self.create_outpoints(node=self.nodes[0], outputs=[{node2_addr: 13}])[0]
```

Comment:
Creating two different txs is intentional here, as otherwise the "Update psbts, should only have data for one input and not the other" test below would fail (if both inputs are part of the same tx, all data is available to both nodes). Added a comment to clarify.